beangulp/importer.py

```diff
@@ -128,3 +128,55 @@ def file_date(self, file) -> Optional[date]:
           (If no date is returned, the file creation time is used. This is the
           default.)
         """
+
+
+class Importer:
+
+    @property
+    def name(self) -> str:
+        return f'{self.__class__.__module__}.{self.__class__.__name__}'
+
+    def identify(self, path: str) -> bool:
```

Comment:
Thoughout the codef I actively avoid using the variable name "path" because it clashes with the common idiom of "from os import path". Prefer "filename"?

Reply:
Given the context I don't think it makes a big difference, but I will rename. I try to avoid confusion between "file name" intended as the base path, and the complete file path. Thus I will probably name this filepath.